Readme.md

```diff
@@ -1,6 +1,6 @@
 # EDK II Project
 
-A modern, feature-rich, cross-platform firmware development environment

```

Comment:
Review comment attached to a single line of code.
Not sure how the buttons differ, "add single comment" vs. "start a review".